ieeeuoft/makeuoft-site-2021
# Changed app start date to Jan 8 in settings and FAQ

## Overview

- Resolves #17 


## Unit Tests Created

- N/A


## Steps to QA

- Read the FAQ to make sure theres no mention of Jan 7th
- Make sure the countdown on the landing shows the number of days between today and Jan 8th
